tests/CodeBehind/UnitTests/BuildTests.cs

```diff
@@ -151,7 +151,7 @@ static BuildTests_CodeBehindBuildTests ()
 		{
 			TestProjectRootDirectory = Path.GetFullPath (Path.Combine (XABuildPaths.TopDirectory, "tests", "CodeBehind", "BuildTests"));
 			CommonSampleLibraryRootDirectory = Path.GetFullPath (Path.Combine (XABuildPaths.TopDirectory, "tests", "CodeBehind", CommonSampleLibraryName));
-			TestOutputDir = Path.Combine (XABuildPaths.TestOutputDirectory, "CodeBehind");
```

Comment:
Should this (and everywhere else) instead be `"temp", "CodeBehind"`?  Otherwise it might just use `temp` directly, which would get confusing?